test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -469,3 +478,35 @@
                  :state "running"}
                 (dissoc body :status)))
          (is (map? (get-in body [:status :experimental :jvm-metrics]))))))))
+
+(deftest status-debug-logging-test
+  (testing "status service logs debug data when setting is enabled"
+    (with-test-logging
+     (with-logger-event-maps
```

Comment:
Do you need `with-test-logging` if you are using `with-logger-event-maps`?  I thought maybe the `with-test-logging` would be unnecessary in that case, but not sure.